Sources/DeepLearning/Operators.swift

```diff
@@ -269,9 +256,11 @@ extension Tensor where Scalar: TensorFlowFloatingPoint {
             )
         })
     }
+    
 }
 
 public extension Tensor where Scalar: FloatingPoint {
+    
```

Comment:
Remove unnecessary empty line. The Google Swift Style Guide neither encourages or discourages the inclusion of an empty line in the beginning and the end of a type declaration, but the choice in our code bases is to not to add them. Same for other places.